python/requirements/ml/requirements_tune.txt

```diff
@@ -43,7 +43,7 @@ transformers==4.19.1; python_version > '3.6'
 # Since we pin ax-platform at an earlier version,
 # we need to pin this as well to prevent incompatible
 # more recent versions from being used.
```

Comment:
Update comment to explain why 3.7 only? 😄

Reply:
Updated